ci/builders/mac_unopt.json

```diff
@@ -1,14 +1,6 @@
 {
     "builds": [
         {
-            "archives": [
```

Comment:
I think these should not have been getting uploaded. @godofredoc had a PR to remove these awhile back, but it got reverted for a different reason.